src/components/FormGroup.tsx

```diff
@@ -0,0 +1,47 @@
+import classNames from "classnames";
+import PropTypes from "prop-types";
+import React from "react";
+
+import "lbh-frontend/lbh/objects/_form-group.scss/";
+
+/**
+ * The proptypes for {@link FormGroup}.
+ */
+export interface FormGroupProps {
+  id?: string;
+  className?: string;
+  /**
+   * Whether there is an error in the form group or not.
+   */
+  error?: boolean;
+  /**
+   * The form elements
+   */
+  children: React.ReactNode;
+}
+
+/**
+ * A component used to group labels and form controls.
```

Comment:
Can you explain the level of this. That is, is this supposed to wrap a single fieldset? Or is it inside a fieldset? Or does it group fieldsets together? Or is it completely separate from fieldset?